src/routes/v1/school.route.js

```diff
@@ -0,0 +1,116 @@
+const express = require('express');
+const validate = require('../../middlewares/validate');
+const schoolValidation = require('../../validations/school.validation');
+const schoolController = require('../../controllers/school.controller');
+const auth = require('../../middlewares/auth');
+
+const router = express.Router();
+router.route('/').post(auth('teacher'), validate(schoolValidation.createSchool), schoolController.createSchool);
+router.route('/').get(auth('teacher'), validate(schoolValidation.getSchools), schoolController.getSchools);
+router.route('/:schoolId').get(auth('teacher'), validate(schoolValidation.getSchool), schoolController.getSchool);
+
+module.exports = router;
+
+/**
+ * @swagger
+ * tags:
+ *   name: Schools
+ *   description: CRUD for schools, only accessible by teachers
+ */
+
+/**
+ * @swagger
+ * path:
+ *   /schools:
+ *     post:
+ *       summary: Create a new school
+ *       tags: [Schools]
+ *       security:
+ *         - bearerAuth: []
+ *       requestBody:
+ *         required: true
+ *         content:
+ *           application/json:
+ *             schema:
+ *                $ref: '#/components/schemas/School'
+ *       responses:
+ *         "200":
+ *           description: OK
+ *           content:
+ *             application/json:
+ *               schema:
+ *                 type: object
+ *                 properties:
+ *                   school:
+ *                     $ref: '#/components/schemas/School'
+ *         "401":
+ *           $ref: '#/components/responses/Unauthorized'
+ *         "403":
+ *           $ref: '#/components/responses/Forbidden'
+ *         "404":
+ *           $ref: '#/components/responses/NotFound'
+ */
+
+/**
+ * @swagger
+ * path:
+ *   /schools?name={subs}:
```

Comment:
we dont need the parameter here in the path, its already defined in further down in parameters

Reply:
so i should just write like /schools. that's all

Reply:
yes here yes